lib/sprockets/commoner/processor.rb

```diff
@@ -44,20 +47,14 @@ def self.call(input)
       end
 
       attr_reader :include, :exclude, :babel_exclude
-      def initialize(root, include: [root], exclude: [], babel_exclude: [/node_modules/])
+      def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude: [/node_modules/])
```

Comment:
Because there's files in there like the sinon source that contain require calls in their UMD, but it won't work as the referenced libraries aren't included, but rather are in the bundle as globals. Everyone who uses sprockets+commoner+circleci will run into this